server/api/rounds.py

```diff
@@ -74,6 +74,10 @@ def active_targeted_contests(election: Election) -> List[Contest]:
 def is_audit_complete(round: Round):
     if not round.ended_at:
         return None
+    # Don't use risk measurements to determine completion for audits with extra batches and always
+    # complete these audits after one round
+    if is_enabled_sample_extra_batches_by_counting_group(round.election):
+        return True
```

Comment:
have we looked at what happens when we manually reopen a round with this feature? I assume it will be overridden just fine?

Reply:
Did a manual test just to be sure, and yup, reopening still works fine!

Reopening clears `round.ended_at`:

https://github.com/votingworks/arlo/blob/53d57c6d61e5b37708793f129f469de4fbaa8636/server/api/support.py#L462

And that's still the first thing we check in `is_audit_complete`:

https://github.com/votingworks/arlo/blob/2d95d9e4e5717d6af99da0e3f6a4d7c23b535f82/server/api/rounds.py#L74-L76